### PR TITLE
Update settings page design

### PR DESCRIPTION
**Pull request by arianeleonard**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature

## Description

Update settings page design
Fix onboarding continue button pressed color

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [x] Tested on all relevant platforms

## Internal Issue (If applicable):

https://dev.azure.com/nventive/Practice%20committees/_backlogs/backlog/Mobile%20(.Net)/Features/?workitem=252839
https://dev.azure.com/nventive/Practice%20committees/_backlogs/backlog/Mobile%20(.Net)/Features/?workitem=245453

### CODE REVIEW

File: src/app/ApplicationTemplate.UWP/Views/Styles/Controls/ToggleSwitch.xaml [hunk 1]
```diff
@@ -13,42 +13,47 @@
 	
 	<!-- ResourceDictionary containing only 1 item doesn't work if that item refers Material resources. -->
 	<x:Int32 x:Key="ToggleSwitchWorkaround">0</x:Int32>
+	
+	<!-- Default native ToggleSwitch styles -->
 
+	<ios:Style x:Key="MaterialToggleSwitchStyle"
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.UWP/Views/Styles/Controls/ToggleSwitch.xaml:19]: Do we really need to redefine the material styles?

File: src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml [hunk 2]
```diff
@@ -42,25 +42,25 @@
 					 Color="{ThemeResource MaterialOnPrimaryColor}" />
 	<SolidColorBrush x:Key="MaterialOnPrimaryLowBrush"
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml:43]: We should not redefine existing Material brushes. We only need to define the color palette. The brushes are updated automatically.

File: src/app/ApplicationTemplate.Shared/Presentation/Settings/SettingsPageViewModel.cs [hunk 3]
```diff
@@ -35,24 +35,13 @@ public partial class SettingsPageViewModel : ViewModel
 			if (logout == MessageDialogResult.Accept)
 			{
 				await this.GetService<IAuthenticationService>().Logout(ct);
+				await this.GetService<ISectionsNavigator>().SetActiveSection(ct, "Login", () => new LoginPageViewModel(isFirstLogin: false), returnToRoot: true);
 			}
 		});
 
-		public IDynamicCommand NavigateToDiagnosticsPage => this.GetCommandFromTask(async ct =>
```
Comment (carlh98) [inline src/app/ApplicationTemplate.Shared/Presentation/Settings/SettingsPageViewModel.cs:41]: NavigateToDiagnosticPage should still be present

Comment (carlh98) [inline src/app/ApplicationTemplate.Shared/Presentation/Settings/SettingsPageViewModel.cs:41]: NavigateToDiagnosticPage should still be present

File: src/app/ApplicationTemplate.Tests/Presentation/NavigationShould.cs [hunk 4]
```diff
@@ -37,21 +37,10 @@ public async Task NavigateFromSettingsToLoginPage()
...
 			// Act
-			var currentSection = await AssertSetActiceSection<SettingsPageViewModel, LoginPageViewModel>(() => new SettingsPageViewModel(), p => p.NavigateToLoginPage, sourceSection);
+			var currentSection = await AssertSetActiceSection<SettingsPageViewModel, LoginPageViewModel>(() => new SettingsPageViewModel(), p => p.Logout, sourceSection);
 
 			// Assert
 			Assert.NotEqual(sourceSection, currentSection);
 		}
-
-		[Fact]
-		public async Task NavigateToDiagnosticsPageAndBack()
```
Comment (carlh98) [inline src/app/ApplicationTemplate.Tests/Presentation/NavigationShould.cs:47]: Should not be removed

Comment (carlh98) [inline src/app/ApplicationTemplate.Tests/Presentation/NavigationShould.cs:47]: Should not be removed

File: src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml [hunk 5]
```diff
@@ -14,157 +14,26 @@
...
-					 Color="{ThemeResource MaterialOnSecondaryColor}"
-					 Opacity="0.38" />
-
 	<!-- Success brushes -->
 	<SolidColorBrush x:Key="MaterialSuccessBrush"
 					 Color="{ThemeResource MaterialSuccessColor}" />
 	<SolidColorBrush x:Key="MaterialOnSuccessBrush"
 					 Color="{ThemeResource MaterialOnSuccessColor}" />
+	
+	<!-- ToggeSwitch Brushes -->
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml:23]: ToggLe*

File: src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml [hunk 6]
```diff
@@ -14,157 +14,26 @@
...
+
+	<!-- Toggle On -->
+	<StaticResource x:Key="MaterialToggleSwitchOnBackgroundBrush"
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml:26]: I do not see the added toggle StaticResource used in this PR, still needed?

Comment (arianeleonard) [reply src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml:26]: Yes they are used in the material toggle switch style. It overrides some of the colors defined here : https://github.com/unoplatform/Uno.Themes/blob/86da421f6e91ffe4831356e41ab7f3ee55d6373b/src/library/Uno.Material/Styles/Controls/v1/ToggleSwitch.Base.xaml

File: src/app/ApplicationTemplate.UWP/Views/Styles/Controls/CommandBar.xaml [hunk 7]
```diff
@@ -126,7 +128,7 @@
 		<Setter Property="Height"
 				Value="56" />
 		<Setter Property="Background"
-				Value="Transparent" />
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.UWP/Views/Styles/Controls/CommandBar.xaml:129]: Why create a static resource instead of using Transparent?

**Review by carlh98 — APPROVED** [review]

**Pull request merged**